generated/niscope/session.py

```diff
@@ -1114,7 +1114,7 @@ class _SessionBase(object):
     Timebase Rate <pSampleClockTimebaseRate.html>`__ and the actual
     sample rate, which can be higher. This property can be used in
     conjunction with the `Sample Clock Timebase Divisor
-    Property <pniscope_SampleClockTimebaseDivisor.html>`__.
```

Comment:
This is the only case I see where it incorrectly tries to match and create a link. Is this a showstopper for this PR?

Reply:
No, it fixes more than it breaks so leaves us in better shape.
Why is this happening? Is it due to bad source data?